docs/sphinx/source/api.rst

```diff
@@ -20,9 +20,10 @@ corresponding procedural code.
 
    location.Location
    pvsystem.PVSystem
+   pvsystem.Array
    tracking.SingleAxisTracker
    modelchain.ModelChain
-
+   modelchain.ModelChainResult
```

Comment:
We forgot to update the [ModelChain section of api.rst](https://pvlib-python--1076.org.readthedocs.build/en/1076/api.html#modelchain)

Perhaps replace the `Attributes` subsection with a `Results` subsection containing this class

Reply:
I think `Attributes` should stay, since it lists things like `system` and `location`. I'll add a new `Results` section after `Running`.

Reply:
Hm looking more closely now,`Attributes` and `Properties` are somewhat redundant and incomplete. We should clean those up too, but doesn't have to be now.